python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -444,6 +444,9 @@ def __get__(self, obj, cls):
 
         # Since we couldn't find the information in the cache, we will need to fully load the object
 
+        if name in obj.__dict__:
+            return obj.__dict__[name]
```

Comment:
I've just written a lazy-loading test to make sure this isn't needed, there was a way that this was broken while I was doing some debugging. It shouldn't be in develop and it's not now but and I don't have time to go back and re-generate what I did to break this. I'm just going to add a unittest to check that the descriptor is never called when accessing a Node object attribute.
I'll upload the test and reversion in a few min